### PR TITLE
Alexan Martin-Eichner

### PR DESCRIPTION
**Pull request by alexanme**
https://github.com/alexanme/hw01/blob/master/README.md

My initial instinct was to edit the readme with vim, since this was just a text editing job and didn't necessarily have anything to do with R, but then I spent a ton of time obsessing over exactly what markdown plugin to use for vim and how to get previews to render, and I realized it might be more productive to obsess over that another day and just do the assignment in R studio.

i couldn't get the alt-headers to work? or at least they didnt display with an underline when i previewed the html versions in rstudio. oh well

I thought i was doing something terribly wrong when I was trying to specify my filepath for my first image, and spent an unnecessary amount of time researching nuances of paths in R markdown, until i realized that i had just typed ".jpg" instead of ".JPG"

>.<

### CODE REVIEW

**Comment by alexanme** [comment]
oh my god i tried to type a ">.<" emoticon and it rendered the ">" as a blockquote

**Comment by lesliesmei** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |        x   |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |        x   |              |            |

Remarks:

* This is so fancy!! You not only embedded a picture of yourself but many graphs.
* Excellent use of line breaks, headers, bold and italic, and bullet points (numbered and unnumbered).
* I don't really know what to criticize.
* I think it's pretty cool you're an Aquarius? I'm a Capricorn-Aquarius cusp.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,64 @@
...
+
+##### (life story cont.)
+
+I wanted a research-related MA so I could have a fancy stamp on my resume making it slightly easier to get industry research jobs. I decided to settle for UChicago, despite my experiences in undergrad, because
+
+1. i already live in hp
+2. i already know how the faculty and administration like to operate
+3. i got a 75% scholarship
+
+And im taking this class because I think some of the most promising mixed-methods in social science today are computational (and it looks good on a resume)!
```
Comment (Yingqiao1) [inline README.md:41]: Good point!

**Review by Yingqiao1 — COMMENTED** [review]
Topic | Excellent | Satisfactory | Needs Work
-- | -- | -- | --
Coding style | x |   |  
Coding strategy | x |   |  
Presentation | x |   |  
Achievement, creativity | x |   |  
Ease of access | x |   |  


Remarks:

* Elaborate on above, especially for "needs work."  This is very nicely down! Very creative! Codes are easy to read, with each forms clearly presented. 
* Some specific praise? 
I love the personality graph, though I don't know how to understand the information in it.  
You have different level of headers, which make it very comfortable to read. 
Different syntax are used fluently, like emphasis of a word in a sentence. 
* Something I learned? I learned what it means to be creative in R
* Specific constructive criticism? You did a great job in presenting nicely in R, but one thing you may have missed is commit, where I only see one. It might be helpful for others to track how you did each step. So you may try to record each crucial step of your coding by writing in commits next time. 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,64 @@
...
+* strong social science division
+* unofficial slogan "where fun goes to die"
+
+Attending UChicago would turn out to be one of the **worst**, or at least most harrowing, decisions of my life. I'm honestly thankful that me and my friends all made it through alive (not everyone can say the same). I got one BA in Mathematics and another BA with honors in Sociology.
+
+---
+
+_Fun fact_: i am obsessed with wacky mixed methods! Check out this directed graph illustrating the system of conditional entailments constructed from a qualitative-comparative coincidence analysis i used in my sociology BA thesis!
+
+![the semantics are kinda complicated][2]
```
Comment (lpstntw) [inline README.md:29]: I love this kind of network illustration!!!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,64 @@
...
+* [books and print comics][4]
+* [music][5]
+* [movies][6]
+
+[1]: figures/me.JPG "this is me"
+[2]: figures/thesisgraph.png "i can explain i promise"
+[3]: figures/birthchart.png "yes i have a stellum in aquarius AND capricorn, its a wild ride"
+[4]: https://www.goodreads.com/user/show/20850992-alexan-martin-eichner "goodreads"
+[5]: https://rateyourmusic.com/~alexanme "rateyourmusic"
+[6]: https://letterboxd.com/alexanme/ "letterboxd"
```
Comment (lpstntw) [inline README.md:64]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            | x         |              |            |
| **Coding strategy**         | x         |              |            |
| **Presentation**            | x         |              |            |
| **Achievement, creativity** | x         |              |            |
| **Ease of access**          | x         |              |            |

Remarks:
* Great format, very easy to read.
* Lots of interesting pictures; they really draw the attention

Comment (chandlerjames) [inline README.md:64]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      X     |              |            |
| **Coding strategy**         |     X      |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |     X      |              |            |
| **Ease of access**          |      X     |              |            |

Remarks:

* This is excellent code
* I enjoyed reading your biography, especially that you voluntarily decided to go to UChicago for undergrad... me too!
* I don't really have any criticisms

**Comment by cysakamoto** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |  x         |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |    x       |              |            |

Great biography page! Let us know if you need something from your teaching staff!

**Pull request closed without merging**